Classes/Xclass/DefaultSplitButton.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace T3kit\themeT3kit\Xclass;
+
+use TYPO3\CMS\Backend\Template\Components\Buttons\InputButton;
+use TYPO3\CMS\Backend\Template\Components\Buttons\SplitButton;
+use TYPO3\CMS\Core\Authentication\BackendUserAuthentication;
+
+class DefaultSplitButton extends SplitButton {
+
+    protected $defaultButtonSet = FALSE;
+
+    /**
+     * Returns the current button
+     *
+     * @return array
+     */
+    public function getButton() {
+        if ($this->defaultButtonSet) {
+            return $this->items;
+        }
+
+        $user = $this->getBackendUser();
+        if (isset($user->user['default_save_button']) && ! empty($user->user['default_save_button'])) {
+            $defaultButtonAction = $user->user['default_save_button'];
+        } else {
+            return parent::getButton();
+        }
+
+        /** @var InputButton $primaryAction */
```

Comment:
@yosokus at this point of the code the phpdoc comment is not valid. The variable is defined null. Move the comment to proper context.